.gitignore

```diff
@@ -0,0 +1,2 @@
+db_data
+.DS_Store
```

Comment:
We should add this files too

vendor/
node_modules/
npm-debug.log
public/storage
public/hot
storage/.key
.env..php
.env.php
.env
bootstrap/compiled.php
app/storage/